src/main/flight/imu.h

```diff
@@ -29,6 +29,7 @@ extern uint32_t accTimeSum;
 extern int accSumCount;
 extern float accVelScale;
 extern int32_t accSum[XYZ_AXIS_COUNT];
+extern bool canUseGPSHeading;
```

Comment:
Is this actually needed?

Reply:
This flag will be toggled by external things (like the GPS rescue mode when we push that up).  Certain phases of rescue mode (like if braking, etc) the heading will be invalid.  This gives it a way to tell the IMU to not trust the GPS heading.